src/m365/pp/commands/dataverse/dataverse-table-row-list.ts

```diff
@@ -0,0 +1,83 @@
+import { Logger } from '../../../../cli/Logger';
+import GlobalOptions from '../../../../GlobalOptions';
+import { powerPlatform } from '../../../../utils/powerPlatform';
+import PowerPlatformCommand from '../../../base/PowerPlatformCommand';
+import commands from '../../commands';
+import request from '../../../../request';
+import { AxiosRequestConfig } from 'axios';
+
+interface CommandArgs {
+  options: Options;
+}
+
+export interface Options extends GlobalOptions {
+  environment: string;
+  name: string;
+  asAdmin?: boolean;
+}
+
+class PpDataverseTableRowListCommand extends PowerPlatformCommand {
+
+  public get name(): string {
+    return commands.DATAVERSE_TABLE_ROW_LIST;
+  }
+
+  public get description(): string {
+    return 'List all row from a specific dataverse table in the specified Power Platform environment.';
```

Comment:
This description is different from the one in the docs. Let's use the one in docs.